Reordered and clarified steps in the doc

kfile opened the pull request:
Instructions were missing several steps in the Azure portal and the organization was out of order.

azurecla commented:
Hi __@benaadams__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

kfile committed:
Reordered and clarified steps in the doc

Instructions were missing several steps in the Azure portal and the organization was out of order.
